pallets/lockdown-mode/src/tests.rs

```diff
@@ -1,12 +1,28 @@
-use super::*;
+use super::{GenesisConfig, *};
 use crate::{mock::*, Error, ACTIVATED, DEACTIVATED};
 use frame_support::{assert_noop, assert_ok, traits::Contains};
 use pallet_balances::{self, Call as BalancesCall};
 use pallet_remark::{self, Call as RemarkCall};
 
+#[test]
+fn genesis_config_default() {
+	let default_genesis = GenesisConfig::default();
+	assert_eq!(default_genesis.activated, ACTIVATED);
+}
+
+#[test]
+fn genesis_config_initialized() {
+	[true, false].into_iter().for_each(|expected| {
+		new_test_ext(expected).execute_with(|| {
+			let lockdown_mode = LockdownModeStatus::<Test>::get();
+			assert_eq!(lockdown_mode, expected);
+		});
+	});
+}
+
 #[test]
 fn activate_lockdown_mode_works() {
-	new_test_ext().execute_with(|| {
+	new_test_ext(false).execute_with(|| {
 		assert_eq!(LockdownModeStatus::<Test>::get(), DEACTIVATED);
```

Comment:
I think this line is not needed since `genesis_config_initialized` already tests that the storage is initialized properly.

Reply:
Done.